base/special/trig.jl

```diff
@@ -740,6 +740,38 @@ mulpi_ext(x::Float32) = DoubleFloat32(pi*Float64(x))
 mulpi_ext(x::Rational) = mulpi_ext(float(x))
 mulpi_ext(x::Real) = pi*x # Fallback
 
+@inline function sinpi_kernel(x::Float64) 
+    x² = x*x
+    x⁴ = x²*x²
+    r  = evalpoly(x², (2.5501640398773415, -0.5992645293202981, 0.08214588658006512,
+					   -0.007370429884921779, 0.0004662827319453555, -2.1717412523382308e-5))
+	
+    return muladd(3.141592653589793, x, x*muladd(-5.16771278004997, 
+				  x², muladd(x⁴, r,  1.2245907532225998e-16)))
+end
+
+@inline function cospi_kernel(x::Float64)
+    x² = x*x
+	r = x²*evalpoly(x², (4.058712126416765, -1.3352627688537357, 0.23533063027900392,
```

Comment:
Indentation looks strange in this function.

Reply:
Indentation looks strange throughout, actually, e.g., around (new) line 800 etc.

Reply:
oops, tabs got into here by accident.